zerver/webhooks/linear/doc.md

```diff
@@ -0,0 +1,18 @@
+Get Linear notifications in Zulip!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url.md!}
+
+    You can refer to Linear's documentation for [webhook events](https://developers.linear.app/docs/graphql/webhooks).
+
+1. Go to your team on Linear and open **Settings**. Select **API**,
+   now click on **New Webhook**.
```

Comment:
Let's make this "... Select **API**, and click on **New Webhook**."

Reply:
Made the change, and now it looks like this:

![image](https://user-images.githubusercontent.com/35286603/232674980-750932f1-3552-4cf9-84ab-fb6c700e80ab.png)